packages/create-expo-app/src/Update.ts

```diff
@@ -0,0 +1,29 @@
+import chalk from 'chalk';
+import { execSync } from 'child_process';
+import checkForUpdate from 'update-check';
+
+export function shouldUseYarn(): boolean {
```

Comment:
Any reason why we shouldn't use `@expo/package-manager`'s `isUsingYarn` instead? Or the package manager that was used when invoking this library?

Reply:
`isUsingYarn` is for projects with a `yarn.lock`, `shouldUseYarn` detects if yarn is installed globally.